server/src/main/java/io/crate/types/BitStringType.java

```diff
@@ -69,7 +71,10 @@ public final class BitStringType extends DataType<BitString> implements Streamer
         new EqQuery<BitString>() {
 
             @Override
-            public Query termQuery(String field, BitString value) {
+            public Query termQuery(String field, BitString value, boolean hasDocValues, IndexType indexType) {
+                if (hasDocValues) {
+                    return SortedSetDocValuesField.newSlowExactQuery(field, new BytesRef(value.bitSet().toByteArray()));
```

Comment:
If both docValues and index is available this should use a `IndexOrDocValuesQuery` instead of prioritizing doc-values.